jdi-light/src/main/java/com/epam/jdi/light/common/PropertyValidator.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.jdi.light.common;
+
+import java.security.InvalidParameterException;
+import java.util.Properties;
+
+public class PropertyValidator {
+
+    public static void validateProperties(Properties properties) {
+        properties.forEach((k, v) -> validate(k.toString(), v.toString()));
+    }
+
+    public static void validate(String name, String value) {
+        for (Property p : Property.values()) {
+            if (p.getName().equals(name)) {
+                if (!value.matches(p.getCheck())) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4532609)